ethereum/rpc/types/utils.go

```diff
@@ -47,12 +47,12 @@ func EthBlockFromTendermint(clientCtx client.Context, queryClient *QueryClient,
 
 	req := &evmtypes.QueryBlockBloomRequest{}
 
-	res, err := queryClient.BlockBloom(ContextWithHeight(block.Height), req)
+	blockBloomResp, err := queryClient.BlockBloom(ContextWithHeight(block.Height), req)
 	if err != nil {
```

Comment:
log the error here (Debug)

Reply:
yeah the reason I did not log there is that we can't have access to the logger easily in the utils.

Another thing, this util function seems not to used anywhere.

Reply:
if it's not used we can remove it